### PR TITLE
fix: check kind installed

### PR DESCRIPTION
**Pull request by troian**

### CODE REVIEW

File: _run/kube/kind.mk [hunk 1]
```diff
@@ -12,6 +13,10 @@ PROVIDER_ENDPOINT ?= http://$(PROVIDER_HOST)
 
 INGRESS_CONFIG_PATH ?= https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
 
+ifeq (, $(shell which kind))
+$(error "No kind in $(PATH), consider install?")
```
Comment (boz) [inline _run/kube/kind.mk:17]: "run 'make devdeps-install' in the top-level directory to install"

Comment (troian) [reply _run/kube/kind.mk:17]: maybe put `devdeps-install` into some mk and include in any makefile that needs it?

Comment (boz) [reply _run/kube/kind.mk:17]: it only really needs to be run once.  i don't care all that much.

**Comment by boz** [comment]
The instructions now don't point to the version pinned by `tools.go` and `go.sum`.

**Pull request closed without merging**